csrankings-b.csv

```diff
@@ -910,7 +910,7 @@ Byron C. Wallace,Northeastern University,http://www.byronwallace.com,KTzRHmwAAAA
 Byron Choi,Hong Kong Baptist University,http://www.comp.hkbu.edu.hk/~bchoi,S6L1eOMAAAAJ
 Byron J. Gao,Texas State University,http://cs.txstate.edu/~jg66,HwB09lAAAAAJ
 Byron J. Williams,Mississippi State University,http://www.fsnhp.msstate.edu/docs/staff/Byron%20%20Williams%20CV%20July%202012%20.pdf,t4bGLWUAAAAJ
-Byung Chul Tak,Kyungpook National University,http://sites.google.com/view/knudiscolab,-wM1-88AAAAJ
+Byung-Chul Tak,Kyungpook National University,http://sites.google.com/view/knudiscolab,-wM1-88AAAAJ

```

Comment:
This name *and* the alias "Byungchul Tak" must be included.

Reply:
Added the alias accordingly.